src/utils/utils.ts

```diff
@@ -1,57 +1,3 @@
-import featuredItemsData from "../data/featureditems.json";
-import appConfig from "../../appConfig";
-let featuredItemArray = [];
-
-/**
- * Sets global featuredItemArray so other functions can use it.
- **/
-export const setFeaturedItemArray = () => {
-  if (featuredItemArray.length !== 0) {
-    // The global array already has what it needs.
-    return;
-  }
-
-  // If it's empty, now is the time to generate and update the array.
-  let itemsArray = [];
-  for (let i = 0; i < featuredItemsData.featuredItems.images.length; i++) {
-    itemsArray.push(featuredItemsData.featuredItems.images[i].split(".")[0]);
-  }
-  // set featuredItemArray
-  featuredItemArray = itemsArray;
-
-  return featuredItemArray;
-};
-
-/**
- * Returns a random image ID from the list of featured items.
- */
-export const generateRandomImageID = () => {
-  if (featuredItemArray.length === 0) {
-    setFeaturedItemArray();
-  }
-  const randomIndex = Math.floor(Math.random() * featuredItemArray.length);
-  return featuredItemArray[randomIndex];
-};
-
-/**
- * Returns a valid featured imageID.
- * Checks if an imageID passed by as a query parameter is included in the pre-approved list of image IDs for featured items.
- * If the imageID is not valid, the function returns a random imageID.
- * @param {string} imageID - optional imageID to check against list of image IDs for featured items.
- */
-export const featuredImageID = (imageID = "") => {
-  if (featuredItemArray.length === 0) {
-    setFeaturedItemArray();
-  }
-  if (imageID !== "") {
-    return featuredItemArray.includes(imageID)
-      ? imageID
-      : generateRandomImageID();
-  } else {
-    return generateRandomImageID();
-  }
-};
-
```

Comment:
These files were deleted [here](https://github.com/NYPL/digital-collections/pull/83/files#diff-5acab41990585fa68ae01ff7b2e2915628d19237840c3061b31fcadc1c0f09b8L1):
<img width="898" alt="Screenshot 2024-03-08 at 3 24 26 PM" src="https://github.com/NYPL/digital-collections/assets/4614468/59861bec-f279-42ff-b6c1-3ea0ec2474f8">

Reply:
I think the back merge [here](https://github.com/NYPL/digital-collections/pull/83) is what caused these changes to not show up automatically in this PR (i'm not 100% sure, but that would be my guess):
<img width="905" alt="Screenshot 2024-03-08 at 3 25 07 PM" src="https://github.com/NYPL/digital-collections/assets/4614468/7b0aa0d9-b147-4c3a-b6bc-b0d825512890">